doc/classes/SubViewportContainer.xml

```diff
@@ -20,6 +20,10 @@
 		</method>
 	</methods>
 	<members>
+		<member name="consume_drag_and_drop" type="bool" setter="set_consume_drag_and_drop" getter="is_consume_drag_and_drop_enabled" default="false">
+			If [code]false[/code], the [SubViewportContainer] is not available as drop location in drag-and-drop operations, but all of the [Control] nodes inside its [Viewport] children.
```

Comment:
```suggestion
			If [code]false[/code], the [SubViewportContainer] is not available as a drop location in drag-and-drop operations, but all of the [Control] nodes inside its [Viewport] children.
```
Missing word?

Also, why is the `false` case listed before the `true` case? I think most of the time we list the `true` case first, but I'm not sure if it's a hard rule

Reply:
`false` is the default value of the parameter, so I put its description first. If the other order would make more sense, I am happy to change it.

Reply:
That makes sense, and there is some precedent for putting `false` first. See what everyone else thinks